scripts/grafana-server/start-server

```diff
@@ -23,8 +23,9 @@ fi
 echo starting server
 
 cp -r ./bin $RUNDIR
-cp -r ./public $RUNDIR
 cp -r ./tools $RUNDIR
+ln -s $(realpath ./public) $RUNDIR
```

Comment:
When I first saw this I wondered if there was a reason why this is copying and not creating a link here. We should research this further.

Reply:
Could track this down to [this](https://github.com/grafana/grafana/pull/22772) change four years ago. I don't think there's any particular reason not to use links. This should be fine.